config/environments/development.rb

```diff
@@ -46,6 +39,6 @@
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
   config.assets.debug = true
-  config.serve_static_assets = true;
+  config.serve_static_files = true;
```

Comment:
Inconsistent indentation detected.<br>Do not use semicolons to terminate expressions.